pkg/pillar/iptables/iptables.go

```diff
@@ -120,7 +70,6 @@ func IptableCmd(log *base.LogObject, args ...string) error {
 
 // Ip6tableCmdOut logs the command string if log is set
 func Ip6tableCmdOut(log *base.LogObject, args ...string) (string, error) {
```

Comment:
I think it reads better if we have them separate.
Instead of:
```
IptableCmd("-i", "eth0", ...)
IptableCmd("-i", "eth1", ...)

Ip6tableCmd("-i", "eth2", ...)
```
We would have something like:
```
IptableCmd(true, "-i", "eth0", ...)
IptableCmd(true, "-i", "eth1", ...)

IptableCmd(false, "-i", "eth2", ...)
```
Where `true` would mean "IPv4".
Without looking at the function code it is not clear to a stranger what this boolean argument means.
If we define some constants like `ipv4` and `ipv6`, it will just make it longer:
```
IptableCmd(ipv4, "-i", "eth0", ...)
IptableCmd(ipv4, "-i", "eth1", ...)

IptableCmd(ipv6, "-i", "eth2", ...)
```
I like that the current syntax resembles the iptables CLIs, where `iptables` and `ip6tables` are also separate commands.
Just my personal opinion.

Reply:
Make sense, thanks!